web/static/httprest/connect/error.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
Can't we have a general template for all the error pages?

Reply:
https://linear.app/autokitteh/issue/ENG-677/improvements-in-integration-web-uis